libs/components/colorpicker/src/lib/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -52,16 +52,14 @@ const SKY_COLORPICKER_DEFAULT_COLOR = '#FFFFFF';
 export class SkyColorpickerInputDirective
   implements OnInit, OnChanges, ControlValueAccessor, Validator, OnDestroy
 {
-  public pickerChangedSubscription: Subscription;
-
   /**
    * Creates the colorpicker element and dropdown. Place this attribute on an `input` element
    * or `button` element, wrap the element in a `sky-colorpicker` component, and set the attribute
    * to the instance of the `sky-colorpicker` component.
    * @required
    */
   @Input()
-  public skyColorpickerInput: SkyColorpickerComponent;
+  public skyColorpickerInput!: SkyColorpickerComponent;
```

Comment:
It's possible for this input to be `undefined` or even `''` with the following setups:
```
<input skyColorpickerInput>
<input [skyColorpickerInput]>
```
Should we account for that?